website/docs/r/network_routed_v2.html.markdown

```diff
@@ -40,6 +40,38 @@ resource "vcd_network_routed_v2" "nsxt-backed" {
 }
 ```
 
+## Example Usage (NSX-T backed routed Org VDC network and a DHCP pool)
+
+```hcl
+resource "vcd_network_routed_v2" "parent-network" {
+  name = "nsxt-routed-dhcp"
+
+  edge_gateway_id = data.vcd_nsxt_edgegateway.existing.id
+
+  gateway = "7.1.1.1"
```

Comment:
This script is nor well formatted

Reply:
Reformatted all scripts with `terraform fmt` in this file.